scripts/hudson/narayana.sh

```diff
@@ -757,13 +747,11 @@ function qa_tests_once {
   cp TaskImpl.properties.template TaskImpl.properties
 
   # check to see which orb we are running against:
-  if [ x$orb = x"openjdk" ]; then
-    orbtype=openjdk
-  elif [ x$orb = x"ibmorb" ]; then
+  if [ x$orb = x"ibmorb" ]; then
     orbtype=ibmorb
 	sed -e "s#^  dist#  ${JAVA_HOME}\${file.separator}jre\${file.separator}lib\${file.separator}ibmorb.jar\\\\\\n  \${path.separator}${JAVA_HOME}\${file.separator}jre\${file.separator}lib\${file.separator}ibmorb.jar\\\\\\n  \${path.separator}dist#" TaskImpl.properties > "TaskImpl.properties.tmp" && mv "TaskImpl.properties.tmp" "TaskImpl.properties"
   else
-    orbtype=jacorb
+    orbtype=openjdk
```

Comment:
I wonder if we should coerce this to cause a fail in the script instead if an unsupported orb was provided?

Reply:
Why not :-) +1 I will put this in my next commit